os/arch/arm/src/amebasmart/amebasmart_reboot_reason.h

```diff
@@ -22,6 +22,7 @@
 #include <tinyara/reboot_reason.h>
 
 enum {
+	REBOOT_SYSTEM_RESET_IWDG     = REBOOT_BOARD_SPECIFIC0, /*System reset by IWDG*/
```

Comment:
REBOOT_SYSTEM_RESET_IWDG = REBOOT_BOARD_SPECIFIC6, /*System reset by IWDG*/

Could you please change this also?